tests/prague/eip7692_eof_v1/eip7069_extcall/test_gas.py

```diff
@@ -50,25 +57,29 @@ def gas_test(
 
     sender = pre.fund_eoa(10**18)
 
-    address_baseline = pre.deploy_contract(setup_code + tear_down_code)
-    address_subject = pre.deploy_contract(setup_code + subject_code + tear_down_code)
+    address_baseline = pre.deploy_contract(Container.Code(setup_code + tear_down_code))
+    address_subject = pre.deploy_contract(
+        Container.Code(setup_code + subject_code + tear_down_code)
+    )
+    # GAS, POP, CALL, 7 times PUSH1 - instructions charged for at every gas run
+    gas_single_gas_run = 2 + 2 + 100 + 7 * 3
     address_legacy_harness = pre.deploy_contract(
         code=(
             # warm subject and baseline without executing
             (Op.BALANCE(address_subject) + Op.POP + Op.BALANCE(address_baseline) + Op.POP)
-            # cold gas run
+            # Baseline gas run
```

Comment:
Why swap the cold and baseline order?

Reply:
It seemed cleaner to keep the baseline at the "bottom of the stack" and then "fetch it" using DUPn 3 times, from varying depths

Reply:
We need to be careful then and ensure setup and teardown have no cold/warm differences.  As long as it's pushes that will hold.  But we need to keep an eye on memory expansion.

Reply:
yeah, I thought about this a bit, but figured if baseline (setup & teardown) has any side effects affecting gas, the test is broken no matter the order, because you subtract the baseline from cold+baseline and warm+basline gas sums, so if any of the "baselines" has different gas cost, the results are off.

Memory expansion is local to the execution frame, so its warmness shouldn't persist between the `Op.CALL`s, but indeed setup/teardown have a limited set of operations they can do.

Reply:
I think it would be a good idea to add these considerations to this function's docstring, just so that anyone using it can have this in mind when designing the setup/teardown.